components/eam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -1330,87 +1330,88 @@ end subroutine aero_model_wetdep
-    real(r8), intent(in) :: del_h2so4_gasprod(:,:) 
-    real(r8), intent(in) :: zm(:,:) 
-    real(r8), intent(in) :: qh2o(:,:) 
-    real(r8), intent(in) :: cwat(:,:)          ! cloud liquid water content (kg/kg)
-    real(r8), intent(in) :: cldfr(:,:) 
-    real(r8), intent(in) :: cldnum(:,:)       ! droplet number concentration (#/kg)
-    real(r8), intent(in) :: vmr0(:,:,:)       ! initial mixing ratios (before gas-phase chem changes)
-    real(r8), intent(inout) :: vmr(:,:,:)         ! mixing ratios ( vmr )
+    !  dummy arguments
+    integer,  intent(in) :: loffset           ! offset applied to modal aero "pointers"
+    integer,  intent(in) :: ncol              ! number columns in chunk
+    integer,  intent(in) :: lchnk             ! chunk index
+    integer,  intent(in) :: latndx(pcols)     ! latitude indices
+    integer,  intent(in) :: lonndx(pcols)     ! longitude indices
+    real(r8), intent(in) :: delt              ! time step size [sec]
+    real(r8), intent(in) :: tfld(:,:)         ! temperature [K]
+    real(r8), intent(in) :: pmid(:,:)         ! pressure at model levels [Pa]
+    real(r8), intent(in) :: pdel(:,:)         ! pressure thickness of levels [Pa]
+    real(r8), intent(in) :: mbar(:,:)         ! mean wet atmospheric mass [amu or g/mol]
+    real(r8), intent(in) :: airdens(:,:)      ! total atms density [molecules/cm**3]
+    real(r8), intent(in) :: invariants(:,:,:) ! invariant density [molecules/cm**3]
+    real(r8), intent(in) :: zm(:,:)           ! midpoint geopotential height above the surface [m]
+    real(r8), intent(in) :: qh2o(:,:)         ! specific humidity [kg/kg]
+    real(r8), intent(in) :: cwat(:,:)         ! cloud liquid water content [kg/kg]
+    real(r8), intent(in) :: cldfr(:,:)        ! cloud fraction [fraction]
+    real(r8), intent(in) :: cldnum(:,:)       ! droplet number concentration [#/kg]
+    real(r8), intent(in) :: vmr0(:,:,:)       ! initial mixing ratios (before gas-phase chem changes) [vmr or mol/mol]
+    real(r8), intent(inout) :: vmr(:,:,:)     ! mixing ratios [vmr or mol/mol]
     type(physics_buffer_desc), pointer :: pbuf(:)
     
     ! local vars 
     
-    integer :: n, m, mm
-    integer :: i,k
+    integer :: imode, mm
+    integer :: icol,kk
     integer :: nstep
 
-    real(r8) :: del_h2so4_aeruptk(ncol,pver)
+    real(r8), pointer :: dgnum(:,:,:)           ! aerosol diameter [m]
+    real(r8), pointer :: dgnumwet(:,:,:)        ! aerosol wet diameter [m]
+    real(r8), pointer :: wetdens(:,:,:)         ! aerosol wet density [kg/m3] 
+    real(r8), pointer :: pblh(:)                ! pbl height [m]
 
-    real(r8), pointer :: dgnum(:,:,:), dgnumwet(:,:,:), wetdens(:,:,:)
-    real(r8), pointer :: pblh(:)                    ! pbl height (m)
-    real(r8)  :: fldcw_all(ncol,pver,gas_pcnst)     ! all gas mass mixing ratio [kg/kg]
+    real(r8), dimension(ncol) :: dvmrdt_col     ! column-integrated tendency from chemistry [kg/kg/m2/s]
```

Comment:
Wouldn't the units for this be [kg/(m2-s)]?  (or more specifically, [kg (mol/mol) / (m2-s)]?